src/controller/place.controller.ts

```diff
@@ -1,68 +1,77 @@
-import { Request, Response } from "express"
+import { Request, Response } from "express";
 import { StatusCode } from "../model/types/StatusCode";
-import { placeAutoComplete, placeHospitalDetails, placeSearchHospital } from "../services/places.service";
+import {
+  placeAutoComplete,
+  placeHospitalDetails,
+  placeSearchHospital,
+} from "../services/places.service";
 
 async function placeAutoCompleteController(req: Request, res: Response) {
```

Comment:
Não é um problema e acho que as alterações aqui foram só do linter, mas pode ser interessante no futuro adicionar uma camada de validação das informações na api (https://jeffsegovia.dev/blogs/rest-api-validation-using-zod).  Além disso, você pode adicionar uma tipagem no `Request` pra não precisar adicionar um _type assertion_.

Ficaria mais ou menos assim:

```suggestion
async function placeAutoCompleteController(req: Request<any, any, { address: string }, any>, res: Response) {
```